src/data/zcl_abapgit_data_deserializer.clas.abap

```diff
@@ -324,6 +325,16 @@ CLASS zcl_abapgit_data_deserializer IMPLEMENTATION.
 
         MOVE-CORRESPONDING ls_file TO ls_result-config. " config file
 
+        " Check if table is included in repo
+        lv_tabfile = replace(
+          val   = ls_file-filename
+          sub   = 'conf.json'
+          with  = 'tabl.xml' ).
```

Comment:
hmm

this would make it very easy to do a "mara.tabl.xml" file, is it possible to check if its already in the system under the repository package?